### PR TITLE
doc: fix meeting minute dates

### PR DESCRIPTION
**Pull request by gibfahn**
Fixes: https://github.com/nodejs/Release/issues/253

cc/  @mjomble, let me know if this LGTY. 

You can check the meeting issues here: https://github.com/nodejs/Release/issues?q=is%3Aissue+is%3Aclosed+label%3Alts-meeting

### CODE REVIEW

**Comment by mjomble** [comment]
In the case of 2017-04-24.md, [2017-03-13.md](https://github.com/nodejs/Release/blob/master/doc/meetings/2017-03-13.md) suggests that April 4 might have been correct and it was the filename that was wrong.
However, I'm not sure, perhaps the meeting was actually later rescheduled for the 24th.

The rest looks good, assuming that these are the correct dates :)

**Comment by mjomble** [comment]
Ah, from #201 it seems like 24 was indeed correct.

**Comment by gibfahn** [comment]
>In the case of 2017-04-24.md, 2017-03-13.md suggests that April 4 might have been correct and it was the filename that was wrong.

Good point, adjusted the `Next meeting` dates too.

File: doc/meetings/2017-06-26.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 - [Github Issue](https://github.com/nodejs/LTS/issues/233)
 - [Meeting Video](https://www.youtube.com/watch?v=xSo6YiKzc5M)
 - [Previous meeting](https://github.com/nodejs/LTS/issues/225)
-- Next meeting: 17 July 2017
+- Next meeting: 19 July 2017
```
Comment (mjomble) [inline doc/meetings/2017-06-26.md]: Should be 19 September?

Comment (gibfahn) [reply doc/meetings/2017-06-26.md]: Not sure what I was doing here, it should actually have been the 17th of July, see https://github.com/nodejs/Release/issues/235.

The meeting was cancelled, but we didn't know that at the time.

**Comment by mjomble** [comment]
Looks good now :+1:

**Pull request merged**